spec/system/admin/orders_spec.rb

```diff
@@ -572,27 +572,49 @@
           expect(page).to have_content "Confirmation emails sent for 2 orders."
         end
 
-        it "can bulk print invoices from 2 orders" do
-          page.find("#listing_orders tbody tr:nth-child(1) input[name='bulk_ids[]']").click
-          page.find("#listing_orders tbody tr:nth-child(2) input[name='bulk_ids[]']").click
-
-          page.find("span.icon-reorder", text: "ACTIONS").click
-          within ".ofn-drop-down .menu" do
-            expect {
-              page.find("span", text: "Print Invoices").click # Prints invoices in bulk
-            }.to enqueue_job(BulkInvoiceJob).exactly(:once)
+        context "can bulk print invoices" do
+          def extract_pdf_content
+            pdf_href = page.all('a').pluck('href')
+            page.find(class: "button", text: "VIEW FILE").click
+            invoice_file_number = pdf_href[0][45..59]
+            invoice_path = "tmp/invoices/#{invoice_file_number}.pdf"
+            reader = PDF::Reader.new(invoice_path)
+            invoice_content = reader.pages.map(&:text)
           end
 
-          expect(page).to have_content "Compiling Invoices"
-          expect(page).to have_content "Please wait until the PDF is ready " \
-                                       "before closing this modal."
+          it "bulk prints invoices in pdf format" do
+            page.find("#listing_orders tbody tr:nth-child(1) input[name='bulk_ids[]']").click
+            page.find("#listing_orders tbody tr:nth-child(2) input[name='bulk_ids[]']").click
+
+            page.find("span.icon-reorder", text: "ACTIONS").click
+            within ".ofn-drop-down .menu" do
+              expect {
+                page.find("span", text: "Print Invoices").click # Prints invoices in bulk
+              }.to enqueue_job(BulkInvoiceJob).exactly(:once)
+            end
+
+            expect(page).to have_content "Compiling Invoices"
+            expect(page).to have_content "Please wait until the PDF is ready " \
+                                         "before closing this modal."
+
+            # we don't run Sidekiq in test environment, so we need to manually run enqueued jobs
+            # to generate PDF files, and change the modal accordingly
+            perform_enqueued_jobs(only: BulkInvoiceJob)
+
+            expect(page).to have_content "Bulk Invoice created"
 
-          # we don't run Sidekiq in test environment, so we need to manually run enqueued jobs
-          # to generate PDF files, and change the modal accordingly
-          perform_enqueued_jobs(only: BulkInvoiceJob)
+            within ".modal-content" do
+              expect(page).to have_link(class: "button", text: "VIEW FILE", href: /invoices/)
 
-          expect(page).to have_content "Bulk Invoice created"
-          expect(page).to have_link(class: "button", text: "VIEW FILE", href: /invoices/)
+              invoice_content = extract_pdf_content # extracts content do variable invoice_content
```

Comment:
Now you don't need the comment any more. 
```suggestion
              invoice_content = extract_pdf_content
```